profiles/admin.py

```diff
@@ -4,6 +4,7 @@
 
 from django.contrib import admin
 
-from .models import Profile
+from .models import Employment, Profile
```

Comment:
Can you remove the changes to this file? We won't be editing the employment records this way AFAIK